v2/openapi/fb-provider-api.yaml

```diff
@@ -867,66 +867,139 @@ components:
       type: string
       enum:
         - Algorand
+        - Arbitrum
+        - Astar
+        - Aurora
         - Avalanche
+        - Axelar
+        - Base
         - Bitcoin
+        - Bitcoin Cash
+        - Bitcoin Cash ABC
+        - Bitcoin SV
+        - Bitkub Chain
         - BNB
+        - Canto
         - Cardano
-        - Casper
+        - Celestia
         - Celo
+        - Chiliz Chain
+        - Chiliz Legacy Chain
+        - Cortex
         - Cosmos
-        - Cronos
+        - Dash
         - Dogecoin
+        - DYDX
+        - eCash
+        - EOS
         - Ethereum
         - Ethereum Classic
+        - EthereumPoW
+        - Evmos
+        - Fantom Opera
+        - Flare
         - Hedera
+        - HT Chain
+        - KAVA
+        - Kusama
         - Litecoin
-        - Monero
-        - Polygon PoS
+        - Moonbeam
+        - Moonriver
+        - NEAR Protocol
+        - NEM NIS1
+        - Oasys
+        - Optimism
+        - Osmosis
+        - Palm
         - Polkadot
+        - Polygon PoS
+        - Ronin
+        - RSK
+        - Shimmer
         - Solana
+        - Songbird Canary-Network
         - Stellar
+        - SX Network
+        - Terra 2.0
+        - Terra Classic
         - Tezos
+        - TokenEX
         - TRON
+        - Velas
+        - WEMIX
+        - XDB Chain
+        - XinFin Network
         - XRP
+        - ZCash
 
-    Layer1Cryptocurrency:
+    CryptocurrencySymbol:
```

Comment:
It should be "cryptocurrency code" or something like that.

For example, for USD:
- Name: United States dollar
- Symbol: $
- Currency code: USD

Reply:
Since this might introduce a breaking change into the API, let's keep it as it is 😞